apps/services/auth-public-api/src/app/modules/delegations/meDelegations.controller.ts

```diff
@@ -254,11 +237,9 @@ export class MeDelegationsController {
   @Scopes(AuthScope.writeDelegations)
   @FeatureFlag(Features.customDelegations)
   @Delete(':delegationId')
-  @HttpCode(204)
```

Comment:
FYI: This is not Swagger related decorator, this actually changes the response status of the operation. I find this a nice feature to let the new `@Documentation` decorator handle this in relation to the documented response status. But we should maybe document this behaviour and usage in the `Nest/Swagger` library's `README`, as all our operation should either way use this new decorator, and therefore use the `response` property of that to control this.